src/pages/library/datasetBuilder/DatasetBuilder.test.ts

```diff
@@ -150,4 +145,22 @@ describe('DatasetBuilder', () => {
     expect(getByLabelText('concept set 1').getAttribute('aria-checked')).toBeTruthy();
     expect(getByLabelText('concept set 2').getAttribute('aria-checked')).toBeTruthy();
   });
+
+  it('shows the home page by default', async () => {
+    render(h(DatasetBuilderView));
```

Comment:
please insert the // Arrange, // Act. and // Assert section headers in tests like these here to match recommended AAA pattern seen in other unit tests.

Reply:
Sure. I'm familiar with AAA from a while ago but as I've practiced it, the comments aren't required. Here's one persons's idea about when AAA comments are necessary. https://blog.ploeh.dk/2013/06/24/a-heuristic-for-formatting-code-according-to-the-aaa-pattern/

Reply:
To be clear, I think AAA comments don't make the code easier to read unless the test code is long or complicated, in which case it may need to be refactored anyway.